exemplar/src/main/java/com/squareup/exemplar/ExemplarJavaApp.java

```diff
@@ -2,23 +2,25 @@
 
 import misk.MiskApplication;
 import misk.MiskModule;
+import misk.config.MiskConfig;
+import misk.environment.Environment;
 import misk.environment.EnvironmentModule;
 import misk.hibernate.HibernateModule;
 import misk.moshi.MoshiModule;
 import misk.web.WebModule;
 
 public class ExemplarJavaApp {
   public static void main(String[] args) {
+    Environment environment = Environment.fromEnvironmentVariable();
+    ExemplarConfig config = MiskConfig.load(ExemplarConfig.class, "exemplar", environment);
```

Comment:
is this why you need the `@JvmStatic`?

Reply:
Exactly. Otherwise, I'd have to do `MiskConfig.INSTANCE.load` which works, but is obviously more verbose.